process/project-lifecycle-documents/osv-schema_graduation_stage.md

```diff
@@ -0,0 +1,92 @@
+
+### Project adoption
+The project must be able to show adoption by multiple parties, which could be production deployments or substantial use by established open source communities, and demonstrate the value of that adoption to either the end users or the open source community.
+
+The OSV Schema is currently exported by:
+- [AlmaLinux](https://github.com/AlmaLinux/osv-database)
+- [Bitnami Vulnerability Database](https://github.com/bitnami/vulndb)
+- [Chainguard](https://packages.cgr.dev/chainguard/osv/all.json)
+- [Curl](https://curl.se/docs/vuln.json)
+- [GitHub Security Advisories](https://github.com/github/advisory-database)
+- [Global Security Database](https://github.com/cloudsecurityalliance/gsd-database)
+- [Go Vulnerability Database](https://github.com/golang/vulndb)
+- [Haskell Security Advisories](https://github.com/haskell/security-advisories)
+- [LoopBack Advisory Database](https://github.com/loopbackio/security/tree/main/advisories)
+- [Malicious Packages Repository](https://github.com/ossf/malicious-packages)
+- [Mageia Advisories](https://advisories.mageia.org/)
+- [OSS-Fuzz](https://github.com/google/oss-fuzz-vulns)
+- [OSV.dev maintained converters](https://google.github.io/osv.dev/data/#converted-data) (Debian, Alpine, NVD)
+- [PyPI Advisory Database](https://github.com/pypa/advisory-database)
+- [Python Software Foundation Database](https://github.com/psf/advisory-database)
+- [RConsortium Advisory Database](https://github.com/RConsortium/r-advisory-database)
+- [Red Hat](https://security.access.redhat.com/data)
+- [Rocky Linux](https://distro-tools.rocky.page/apollo/openapi/#osv)
+- [Rust Advisory Database](https://github.com/RustSec/advisory-db)
+- [SUSE](https://www.suse.com/support/security/)
+- [Ubuntu](https://github.com/canonical/ubuntu-security-notices/)
+
+### Release cadence
+The project must be able to show a consistent release cadence.
+  * https://github.com/ossf/osv-schema/releases
+
+### Governance
+Projects must have documented project governance and be able to demonstrate that governance in action.
+  * https://github.com/ossf/osv-schema/blob/main/CHARTER.md
+
+Have a defined and documented roadmap and annual goals for the project
+  * "link to roadmap and goals"
+
+Project has met at least 4 times over a period of at least 2 months since becoming incubating
+  * The project has a standing agenda item in the Vulnerability Disclosures Working Group meetings. [Meeting Notes](https://docs.google.com/document/d/1TdxiFofLOfpHUEQILlKq7qkjSsRXVab0uApSDJ8c5rI/edit?tab=t.0)
+
+Implements, practices, and refines mature software development and release practices, such as adherence to semantic versioning, and having a declared policy for stable releases and backported fixes.
+  * "link to policy for (or describe here) software development and release practices"
```

Comment:
@oliverchang can you suggest an edit here to describe the release process of the schema?

Reply:
Actually, it'd be better for that to be in something like a RELEASING.md in the osv-schema repo, I think.

Reply:
I will take a look at making a RELEASING.md file, the current release process is pretty simple:
- Bump up the version (patch version bump for new ecosystems, minor version bump for non breaking schema field changes)
- Add schema changes to the changelog
- Publish github release
- Update the github pages branch to the new release.

Reply:
Done!